src/main/eo/org/eolang/math/number.eo

```diff
@@ -23,6 +23,9 @@
 +package org.eolang.math
 +home https://github.com/objectionary/eo-math
 +version 0.0.0
++alias org.eolang.math.positive-infinity
++alias org.eolang.math.nan
++alias org.eolang.switch
 +rt jvm org.eolang:eo-math:0.0.0
 
 [n] > number
```

Comment:
@maxonfjvipon I guess `number` should decorate `n`, e.g. 
```
[n] > number
  n > @
```

Reply:
done